ibm_catalog.json

```diff
@@ -17,138 +17,124 @@
-                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/documentation/main/icons/ModuleExampleDiagram.svg",
-                                    "type": "image/svg+xml"
-                                },
-                                "description": "This is an example of creating 2 MZR clusters in same VPC, and deploying the observability agents in the clusters."
-                            }
-                        ]
-                    }
-                },
-                {
-                    "label": "Cluster security group rules",
-                    "name": "add_rules_to_sg",
-                    "working_directory": "examples/add_rules_to_sg",
-                    "architecture": {
-                        "diagrams": [
-                            {
-                                "diagram": {
-                                    "caption": "Cluster security group rules example",
-                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/documentation/main/icons/ModuleExampleDiagram.svg",
-                                    "type": "image/svg+xml"
-                                },
-                                "description": "This example will add security rules to the `kube-<vpcid>` and `kube-<clusterId>` security groups."
-                            }
-                        ]
-                    }
-                },
-                {
-                    "label": "Cross account KMS encryption",
-                    "name": "cross_kms_support",
-                    "working_directory": "examples/cross_kms_support",
-                    "architecture": {
-                        "diagrams": [
-                            {
-                                "diagram": {
-                                    "caption": "Cross account KMS encryption example",
-                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/documentation/main/icons/ModuleExampleDiagram.svg",
-                                    "type": "image/svg+xml"
-                                },
-                                "description": "A single zone OCP VPC cluster configured with KMS encryption for cluster data and worker block storage using a KMS encryption key from another account."
-                            }
-                        ]
+                  "label": "quickstart",
+                  "name": "quickstart",
+                  "install_type": "fullstack",
+                  "working_directory": "solutions/quickstart-vpc",
+                  "configuration": [
+                    {
+                      "key": "ibmcloud_api_key"
+                    },
+                    {
+                      "key": "prefix",
+                      "required": true
+                    },
+                    {
+                      "key": "use_existing_resource_group"
```

Comment:
force this to true so it lives beside the resource_group_name inputs in the UI (see internal [issue](https://github.ibm.com/GoldenEye/issues/issues/12542))